index.js

```diff
@@ -0,0 +1,16 @@
+const mobileNav = document.querySelector('.mobile-nav');
+const hamburgerMenu = document.querySelector('.hamburger');
+const hamburgerClose = document.querySelector('.hamburger-close');
+const mobileNavLink = document.querySelectorAll('.mobile-nav-link');
+
+hamburgerMenu.addEventListener('click', () => {
+  mobileNav.classList.toggle('is-active');
+  hamburgerClose.classList.toggle('is-active');
+});
+
+mobileNavLink.forEach((link) => {
+  link.addEventListener('click', () => {
+    mobileNav.classList.toggle('is-active');
+    hamburgerClose.classList.toggle('is-active');
+  });
+});
```

Comment:
When I click on the `Portfolio` link in the mobile menu, I am not redirected to the correct section on the page. The reason for that is because the section has a `works` id but the link redirects to `work`.

<img width="182" alt="Screen Shot 2022-07-18 at 8 40 59" src="https://user-images.githubusercontent.com/82426682/179524328-ff0ea250-eb8c-4537-881e-dadc6a652ddf.png">

<img width="245" alt="Screen Shot 2022-07-18 at 8 41 23" src="https://user-images.githubusercontent.com/82426682/179524403-7ff37595-7c44-45bc-8537-1bdea86506e6.png">

Please fix it.

<img width="663" alt="Screen Shot 2022-07-18 at 8 41 57" src="https://user-images.githubusercontent.com/82426682/179524500-2f11426d-6616-45c0-bf01-481e6be57cb4.png">